src/Analyzers/CSharp/Tests/UseUTF8StringLiteral/UseUTF8StringLiteralTests.cs

```diff
@@ -0,0 +1,1388 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.UseUTF8StringLiteral;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseUTF8StringLiteral
+{
+    using VerifyCS = CSharpCodeFixVerifier<
+        UseUTF8StringLiteralDiagnosticAnalyzer,
+        UseUTF8StringLiteralCodeFixProvider>;
+
+    public class UseUTF8StringLiteralTests
+    {
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseUTF8StringLiteral)]
+        public async Task TestNotInAttribute()
```

Comment:
Heads up that the design may change to allow UTF8 strings in attributes.

Reply:
I am subscribed to the issue, and my finger is hovering over the delete key!